src/robusta/core/sinks/incidentio/incidentio_sink.py

```diff
@@ -0,0 +1,115 @@
+import logging
+from typing import Optional, Dict, List, Any
+from urllib.parse import urljoin
+from robusta.core.sinks.incidentio.incidentio_client import IncidentIoClient
+from robusta.core.sinks.incidentio.incidentio_api import AlertEventsApi
+
+import requests
+
+from robusta.core.reporting.base import BaseBlock, Finding, FindingSeverity, Enrichment, Link, LinkType
+from robusta.core.reporting.blocks import (
+    HeaderBlock,
+    JsonBlock,
+    LinksBlock,
+    ListBlock,
+    MarkdownBlock,
+    TableBlock,
+    KubernetesDiffBlock,
+)
+from robusta.core.sinks.sink_base import SinkBase
+from robusta.core.sinks.sink_config import SinkConfigBase
+from robusta.core.sinks.incidentio.incidentio_sink_params import IncidentioSinkParams
+
+
+class IncidentioSink(SinkBase):
+    def __init__(self, sink_config: SinkConfigBase, registry):
```

Comment:
I think the `sink_config` type here should be the concrete type, `IncidentioSinkConfigWrapper`
The base class doesn't have the attribute `incidentio_sink`

Reply:
Yeah, you are right, I've updated my implementation.